src/openms/source/ANALYSIS/OPENSWATH/DIAHelper.cpp

```diff
@@ -413,6 +486,19 @@ namespace OpenMS::DIAHelpers
       }
     }
 
+    void simulateSpectrumFromSequence(const NASequence& na,
+                                        std::vector<double>& first_isotope_masses, //[out]
+                                        std::vector<std::pair<double, double> >& isotope_masses, //[out]
+                                        NucleicAcidSpectrumGenerator const * generator, int charge)
+    {
+      getTheorMasses(na, first_isotope_masses, generator, charge);
+      for (std::size_t i = 0; i < first_isotope_masses.size(); ++i)
+      {
+      getAveragosineIsotopeDistribution(first_isotope_masses[i], isotope_masses,
```

Comment:
haha is this really a name? Averagosine :P

Reply:
My brain was pretty mush by yesterday afternoon...